plugins/packages/redis/lib/manifest.json

```diff
@@ -65,6 +71,17 @@
       "key": "password",
       "type": "password",
       "description": "Enter password"
+    },
+    "tls": {
+      "label": "TLS",
+      "key": "tls",
+      "type": "toggle",
+      "description": "Toggle for TLS"
+    },
+    "connection_options": {
```

Comment:
Since it is only added for tls option, can we rethink this part? Connection options are generally spread at the root level. One approach is just prefix it with `tls_*`. 

But instead of that, I think we can do it like how we've done it for other datasources. Please have a look at the manifest file for Postgres datasource for reference.
We can have dropdown for the type of tls selection with `None` and `CA certificate` as an option. That will solve it, isn't it?

Reply:
@akshaysasidrn I have followed the postgress plugin, the TLS options for ioredis do not support certificates, at least if you are not using the Redis.Cluster, which it's the case here

Reply:
Could you give me some example of what you mean by prefixing?

Reply:
I'm not quite sure what you mean here, the Connection Options on postgress plugin is shown all the time, it does not control if it will show or not the Connection options.

Reply:
I have changed it, but I don't understand why I need to add the common fields under the tls_certificate section, it should be outside of this section, but if I let outside it doesn't render the UI.


https://github.com/user-attachments/assets/402005c6-a484-49f3-ba84-63188fbf90bd

Reply:
@akshaysasidrn I've understood now. But we should use the connection options inside the TLS section, because we can or not inform HOST/PORT and Family, those parameters are not mandatory but can me informed if needed

Reply:
@xwellingtonx Is that something that blocks you at the moment? If not, we can add it later iteratively.

Reply:
@akshaysasidrn No, it's not! I can remove it so, I just added in case someone else would need after the deployment. I'm going to remove it since I need this be deployed so it unblock my APP

Reply:
Cool lets go ahead then 🙌🏾

Reply:
@akshaysasidrn  all done.

Reply:
@xwellingtonx thanks, will have this tested and then take it forward for a release.